scripts/zones/Windurst_Woods/npcs/Apururu.lua

```diff
@@ -2,205 +2,169 @@
-        player:addKeyItem(dsp.ki.SECOND_DARK_MANA_ORB);
-        player:addKeyItem(dsp.ki.THIRD_DARK_MANA_ORB);
-        player:addKeyItem(dsp.ki.FOURTH_DARK_MANA_ORB);
-        player:addKeyItem(dsp.ki.FIFTH_DARK_MANA_ORB);
-        player:addKeyItem(dsp.ki.SIXTH_DARK_MANA_ORB);
-
-        player:messageSpecial(KEYITEM_OBTAINED,dsp.ki.FIRST_DARK_MANA_ORB);    -- Display the key item messages
-        player:messageSpecial(KEYITEM_OBTAINED,dsp.ki.SECOND_DARK_MANA_ORB);
-        player:messageSpecial(KEYITEM_OBTAINED,dsp.ki.THIRD_DARK_MANA_ORB);
-        player:messageSpecial(KEYITEM_OBTAINED,dsp.ki.FOURTH_DARK_MANA_ORB);
-        player:messageSpecial(KEYITEM_OBTAINED,dsp.ki.FIFTH_DARK_MANA_ORB);
-        player:messageSpecial(KEYITEM_OBTAINED,dsp.ki.SIXTH_DARK_MANA_ORB);
-
-        player:setVar("MissionStatus_orb1",1);    -- Set the orb variables; 1 = not handled; 2 = handled;
-        player:setVar("MissionStatus_orb2",1);
-        player:setVar("MissionStatus_orb3",1);
-        player:setVar("MissionStatus_orb4",1);
-        player:setVar("MissionStatus_orb5",1);
-        player:setVar("MissionStatus_orb6",1);
-    elseif (csid == 143 or csid == 145) then
-
-        finishMissionTimeline(player,1,csid,option);
-
-        player:setVar("MissionStatus_orb1",0);
-        player:setVar("MissionStatus_orb2",0);
-        player:setVar("MissionStatus_orb3",0);
-        player:setVar("MissionStatus_orb4",0);
-        player:setVar("MissionStatus_orb5",0);
-        player:setVar("MissionStatus_orb6",0);
-
-
-        player:delKeyItem(dsp.ki.FIRST_GLOWING_MANA_ORB);    -- Remove the glowing orb key items
-        player:delKeyItem(dsp.ki.SECOND_GLOWING_MANA_ORB);
-        player:delKeyItem(dsp.ki.THIRD_GLOWING_MANA_ORB);
-        player:delKeyItem(dsp.ki.FOURTH_GLOWING_MANA_ORB);
-        player:delKeyItem(dsp.ki.FIFTH_GLOWING_MANA_ORB);
-        player:delKeyItem(dsp.ki.SIXTH_GLOWING_MANA_ORB);
-
-    -- The Kind Cardian
-    elseif (csid == 392 and option == 1) then
-        player:setVar("theKindCardianVar",1);
-    elseif (csid == 397) then
-        player:delKeyItem(dsp.ki.TWO_OF_SWORDS);
-        player:setVar("theKindCardianVar",2);
-        player:addFame(WINDURST,30);
-        player:tradeComplete();
-
-    -- Windurst 8-2
-    elseif (csid == 588) then
-        player:setVar("MissionStatus",1);
-        player:addKeyItem(dsp.ki.MANUSTERY_RING);
-    elseif (csid == 601) then
-        player:setVar("MissionStatus",3);
-    elseif (csid == 590) then
-        player:setVar("MissionStatus",7);
-    elseif (csid == 592) then
-        player:setVar("MissionStatus",9);
-    elseif (csid == 609) then
+    -- WINDURST 1-2: THE HEART OF THE MATTER
+    if csid == 137 then
+        player:setVar("MissionStatus", 1)
+
+        npcUtil.giveKeyItem(player, {dsp.ki.FIRST_DARK_MANA_ORB, dsp.ki.SECOND_DARK_MANA_ORB, dsp.ki.THIRD_DARK_MANA_ORB, dsp.ki.FOURTH_DARK_MANA_ORB, dsp.ki.FIFTH_DARK_MANA_ORB, dsp.ki.SIXTH_DARK_MANA_ORB})
```

Comment:
see https://github.com/DarkstarProject/darkstar/pull/4919#discussion_r191461912
(I was in single commit view when I commented, so github called it outdated)